attribute-service/src/integrationTest/java/org/hypertrace/core/attribute/service/AttributeServiceTest.java

```diff
@@ -76,15 +80,19 @@ public class AttributeServiceTest {
           .build();
 
   private final Map<String, String> requestHeaders = Map.of("x-tenant-id", TEST_TENANT_ID);
+  private final Map<String, String> systemRequestHeaders = Map.of("x-tenant-id", ROOT_TENANT_ID);
 
-  private static AttributeServiceClient client;
+  private static AttributeServiceBlockingStub stub;
```

Comment:
Changes in this file:
1. Replace `client` with `blockingStub` since the `getAttributes()` API is not present in the `client`
2. Update the tests to test the `getAttributes()` API instead of the deprecated `findAttributes()` API
3. Add new tests to test without custom filtering and with custom filtering set to true/false.

Reply:
Makes sense.
No new integration test is needed since we are not adding the ability to filter by "custom".